docs/en/latest/plugins/skywalking.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: skywalking
+title: SkyWalking
```

Comment:
No need for camel naming, keep it lowercase like the rest of the plugin documentation.

Reply:
Zipkin is capitalized and I was trying to keep consistent, I will revert this then.